### PR TITLE
contributing docs updates

### PR DESCRIPTION
**Pull request by Islast**
- [x] I'm ready to merge

* **What's the context for this pull request?** 
_(this is a good place to reference any issues that this PR addresses)_
Issues #110 and #117 

* **What's new?**
updated install instructions as suggested in #110 
following #117 I have added a short section to the development guide in which to include clarifications on style.

* **What should a reviewer feedback on?**
@KirstieJane I like to keep you pinged in any changes to the contributing docs. It would be great if you could take a glance over and make sure they read well.
@wingedRuslan I've added some bullet points to the development guide. I'd like to know: would seeing something like this have eased the concerns you raised in #117? Or was your question a more general one about wanting to know when it was appropriate to change existing code to fit PEP8 style? I'm trying to feed this question back into the developer's guide it would be great to hear your opinion

* **Does anything need to be updated after merge?** 
_(e.g the wiki or the WhitakerLab website)_
nope :)

### CODE REVIEW

**Comment by wingedRuslan** [comment]
@Islast actually I was wanting to know if it was appropriate to change existing code to fit PEP8 style. So it was more a general question. From my perspective, it is unnecessary information to include in the development guide. We will refactor the code(imports +shebangs) so that it won't be confusing to new contributors.

**Comment by Islast** [comment]
Thanks for the feedback Ruslan. I've changed it to simply note that contributors should feel free to update existing code when it breaks PEP8

File: DEVELOPMENT_GUIDE.md [hunk 1]
```diff
@@ -14,14 +14,15 @@ The [development guide](#development-guide) deals with the specifics of contribu
...
-Use `pip install -e git+https://github.com/WhitakerLab/scona.git` to install scona in editable mode. This means that the python install of scona will be kept up to date with any changes you make, including switching branches in git.
+Use `pip install -e git+https://github.com/WhitakerLab/scona.git#egg=scona` to install scona in editable mode. This means that the python install of scona will be kept up to date with any changes you make, including switching branches in git.
 
 ### Linting
 
 scona uses the [PEP8 style guide](https://www.python.org/dev/peps/pep-0008/).
 You can use [flake8](http://flake8.pycqa.org/en/latest/) to lint code.
 We're quite a young project (at time of writing in January 2019) and so we aren't going to be super hardcore about your linting!
 Linting should make your life easier, but if you're not sure how to get started, or if this is a barrier to you contributing to `scona` then don't worry about it or [get in touch](CONTRIBUTING.md#how-to-get-in-touch) and we'll be happy to help you.
+Feel free also to correct unlinted code in scona when you come across it!:sparkles:
```
Comment (KirstieJane) [inline DEVELOPMENT_GUIDE.md]: AMEN! 🙌

**Comment by KirstieJane** [comment]
Ready to merge @Islast 🚀

**Pull request merged**